### PR TITLE
feat: audio child nodes of par tags now optional

### PR DESCRIPTION
**Pull request by dan-mckay**

### CODE REVIEW

File: playground/app.es6 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { parseBook, parseSmil, parseAllSmil } from '../index';
 
-const URI = '/books/xplor-g8-smil';
```
Comment (dariocravero) [inline playground/app.es6:3]: @dmcaodha could you revert the change in playground? :)

**Comment by dan-mckay** [comment]
@dariocravero I reverted the file paths in playground
